webapp/views/Guest/Guest.js

```diff
@@ -6,7 +6,7 @@ import ForgotPasswordView from '@webapp/guest/forgotPassword/forgotPasswordView'
 
 import { guestModules } from '@webapp/app/appModules'
 
-const GuestView = () => (
+const Guest = () => (
```

Comment:
ForgotPasswordView and ResetPasswordView should be moved under webapp/views/guest (and add an index.js)